packages/giselle-engine/src/core/generations/generate-image.ts

```diff
@@ -49,7 +49,7 @@ export function generateImage(args: {
 				throw new Error("Invalid generation type");
 			}
 
-			const langfuse = new Langfuse();
```

Comment:
`generate-image.ts` uses Langfuse directly, but I'd like to make fundamental improvements in a separate PR